### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by loic-sharma**
Created the Windows project using:

```
flutter create --platforms=windows .
```

Demo:

image

### CODE REVIEW

File: samples/flutter_app/windows/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Project-level configuration.
```
Comment (loic-sharma) [inline samples/flutter_app/windows/CMakeLists.txt:1]: Should this have license headers?

Other entry points' in this project do not have license headers.

The Flutter repo's samples do have license headers on their native entry points. For example: https://github.com/flutter/flutter/blob/master/examples/flutter_view/windows/CMakeLists.txt#L1-L3

Comment (natebosch) [reply samples/flutter_app/windows/CMakeLists.txt:1]: If the flutter repo adds copyright headers I don't see any risk in doing the same here.

**Review by natebosch — COMMENTED** [review]
I'm not sure how to review this - is it all generated?

**Comment by loic-sharma** [comment]
> I'm not sure how to review this - is it all generated?

The first two commits are generated by `flutter create --platforms=windows .`, the last commit contains manual edits: https://github.com/google/generative-ai-dart/pull/59/commits/27e1c8ccc34dee72267a55860e24ebf02285da2c

**Review by natebosch — APPROVED** [review]
RSLGTM

**Pull request merged**